### PR TITLE
Test case for tmp dir fix. closes #2047

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

**Comment by kshakir** [comment]
👎  This centaur test only runs on JES. One should instead add a new one that runs on _all_ backends. https://github.com/broadinstitute/cromwell/issues/2047 mentions a partial example:

> … a centaur test … with command:
> 
> command {
>   echo "hello" > tmp
> }

**Comment by ruchim** [comment]
@kshakir re-review

File: src/main/resources/standardTestCases/tmp_dir.test [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: tmp_dir
+testFormat: workflowsuccess
+backends: [Local, Jes]
```
Comment (kshakir) [inline src/main/resources/standardTestCases/tmp_dir.test]: Take this line out. This is still making this test "centaurJes" only, as it's the only test setup with both Local+Jes.

**Comment by kshakir** [comment]
Test is currently Jes only still, but should be a quick fix, then 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/167/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by kshakir** [comment]
Re: "Jes only" comment, this is the line that uses an _and_ instead of an _or_ for "[Local, Jes]": https://github.com/broadinstitute/centaur/blob/54f824eaad85fc42f5214644e2e004ffaa9a1b3c/src/main/scala/centaur/test/standard/StandardTestCase.scala#L23

File: src/main/resources/standardTestCases/tmp_dir/tmp_dir.wdl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+workflow tmp_dir {
+    call mkTmpDir
+}
+
+task mkTmpDir {
+    command {
+        echo "tmp_dir test wdl" > tmp
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/tmp_dir/tmp_dir.wdl]: Do we also want to test the temp dir itself?

`echo "tmp_dir test wdl 2" > $TMPDIR/tmp`

**Comment by cjllanwarne** [comment]
Please verify that we can read the output files too (e.g. `String out = read_string("tmp"), ...`). Then 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/167/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**